core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1055,28 +1055,28 @@ public MetricKey build() {
           .build();
   public static final MetricKey WORKER_BLOCK_REMOVER_TRY_REMOVE_COUNT =
       new Builder("Worker.BlockRemoverTryRemoveCount")
-          .setDescription("The total number of blocks tried to be removed from this worker "
-              + "by asynchronous block remover.")
+          .setDescription("The total number of blocks this worker attempted to remove "
+              + "with asynchronous block remover.")
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
   public static final MetricKey WORKER_BLOCK_REMOVER_REMOVED_COUNT =
-      new Builder("Worker.BlockRemoverBlocksToRemovedCount")
-          .setDescription("The total number of blocks removed from this worker "
+      new Builder("Worker.BlockRemoverBlocksRemovedCount")
+          .setDescription("The total number of blocks successfully removed from this worker "
               + "by asynchronous block remover.")
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
   public static final MetricKey WORKER_BLOCK_REMOVER_TRY_REMOVE_BLOCKS_SIZE =
       new Builder("Worker.BlockRemoverTryRemoveBlocksSize")
-          .setDescription("The size of blocks to be removed from this worker "
+          .setDescription("The number of blocks to be removed from this worker at this moment "
```

Comment:
(nit) also change the metric name?

Reply:
@HelloHorizon This is essentially the queue size (of the to-be-removed) blocks at a moment. I think the metric name makes sense to me so i will leave it as-is.